assets/sass/v2/_yubikey.scss

```diff
@@ -0,0 +1,8 @@
+.yubikey-demo {
+  background: url('../img/icons/mfa/yubikeyDemo.png') no-repeat center center;
+  background-size: contain;
+  width: 100%;
+  min-height: 102px;
+  margin-bottom: 20px;
+  background-color: 'red';
+}
```

Comment:
FYI this is [identical to v1](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/modules/_enroll.scss#L122-L128), just scoping it within our v2 code.